### DISCUSSION TITLE
Next steps

### DISCUSSION THREAD

**Discussion by Quizznor** [shukkkur/VolleyVision#2] [Q&A] [answered]
This looks very interesting! Thanks for your work, great project.

From what I'm understanding, the next steps are implementing rally-level analysis?
Do you have any preferred way for other people to contribute to this repo?

Cheers

**Comment by shukkkur** [comment, marked as answer]
@Quizznor Hi! Thanks for your kind words and opening this discussion! 
Yes, as soon as I finish the court detection and tracking and I will focus on game analysis and statistics, since ultimately that is the goal of the project :) 

As for contribution please check out - [CONTRIBUTING.md](https://github.com/shukkkur/VolleyVision/blob/main/CONTRIBUTING.md)

**Comment by komax74** [comment]
I discovered this beautiful project, but I saw that it hasn't been updated for a few years. Too bad, has the project been discontinued?
